src/templates/static/legal/regulation.jsx

```diff
@@ -562,7 +562,7 @@ const Regulation = () => {
             </Box>
 
             <Box header={it.L('Deriv (V) Ltd')} >
-                <p>{it.L('Deriv (V) Ltd, Govant Building, Port Vila, PO Box 1276, Vanuatu, Republic of Vanuatu. Licensed and regulated by the Vanuatu Financial Services Commission - [_1]view licence[_2].', `<a href=${it.url_for('download/regulation/Vanuatu-license.pdf')} target="_blank">`, '</a>')}</p>
+                <p>{it.L('Deriv (V) Ltd, Govant Building, Port Vila, PO Box 1276, Vanuatu, Republic of Vanuatu. Licensed and regulated by the Vanuatu Financial Services Commission.', `<a href=${it.url_for('download/regulation/Vanuatu-license.pdf')} target="_blank">`, '</a>')}</p>
```

Comment:
Hi @meenakshi-deriv , you can remove the anchor tagged content, I think we don't need that anymore.